src/screens/ShareTrip/ShareTrip.tsx

```diff
@@ -0,0 +1,158 @@
+import { RouteProp } from "@react-navigation/core";
+import { StackNavigationProp } from "@react-navigation/stack";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import {
+    Platform,
+    ScrollView,
+    Share,
+    ShareContent,
+    StyleSheet,
+} from "react-native";
+import { Avatar, Button, Header, Text } from "react-native-elements";
+import SvgLogo from "../../components/SvgLogo";
+import RootStackParamList from "../../types/RootStackParamList";
+import { useCreateInvitationMutation } from "./types/create-invite.mutation";
+
+type ShareTripScreenNavigationProp = StackNavigationProp<
+    RootStackParamList,
+    "ShareTrip"
+>;
+type ShareTripScreenRouteProp = RouteProp<RootStackParamList, "ShareTrip">;
+
+type Props = {
+    navigation: ShareTripScreenNavigationProp;
+    route: ShareTripScreenRouteProp;
+};
+
+export default function ShareTrip(props: Props): JSX.Element {
+    const { t } = useTranslation();
+
+    const trip = props.route.params.trip;
+    const [execute, { error, loading }] = useCreateInvitationMutation();
+
+    const invitationBaseUrl = "https://vacatius.com/invite/";
```

Comment:
I think we should put as an additional variable to https://github.com/vacatius/mobile-app/blob/bab71e080f27888abd9f42d57c535ba68f55c73a/src/get-environment.ts#L6-L13

Eventually there URL might change and we could support multiple environments.

Reply:
Yeah, that's probably a good idea. I've updated the PR accordingly (see 51edee5085ea88e65dbd6a16cc6d8407633371e8).